framework/helpers/BaseHtml.php

```diff
@@ -975,16 +979,13 @@ public static function radioList($name, $selection = null, $items = [], $options
             }
             $index++;
         }
+        $visibleContent = implode($separator, $lines);
 
-        if (isset($options['unselect'])) {
-            // add a hidden field so that if the list box has no option being selected, it still submits a value
-            $hidden = static::hiddenInput($name, $options['unselect']);
-            unset($options['unselect']);
-        } else {
-            $hidden = '';
+        if (false === $tag) {
```

Comment:
This way some developers are avoiding erroneous use `=` instead of `==`|`===`, but I am against this trick.